capstone/src/components/User/Header/Header.jsx

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+import { Link } from 'react-router-dom';
+import './Header.css';
+
+export default function Header({ setIsLoggedIn }) {
+  function handleLogOut() {
+    localStorage.removeItem('current_user_id');
+    axios.defaults.headers.common = {};
```

Comment:
Can you explain what's going on here?

Reply:
So in my last lab with codepath, they did a demo where they used the headers to authenticate the user's information through the headers of the network request made to Parse. Therefore, I had to set it when logged in and remove it when logged out. 
They said it was not a secure practice, but it was good for prototyping
And the localStorage is to help the UI determine the logIn State mainly.